
package com.netsuite.webservices.platform;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.1.0
 * 2015-05-08T11:20:34.168+08:00
 * Generated source version: 3.1.0
 */

@WebFault(name = "asyncFault", targetNamespace = "urn:faults_2014_2.platform.webservices.netsuite.com")
public class AsyncFault extends Exception {
    
    private com.netsuite.webservices.platform.faults.AsyncFault asyncFault;

    public AsyncFault() {
        super();
    }
    
    public AsyncFault(String message) {
        super(message);
    }
    
    public AsyncFault(String message, Throwable cause) {
        super(message, cause);
    }

    public AsyncFault(String message, com.netsuite.webservices.platform.faults.AsyncFault asyncFault) {
        super(message);
        this.asyncFault = asyncFault;
    }

    public AsyncFault(String message, com.netsuite.webservices.platform.faults.AsyncFault asyncFault, Throwable cause) {
        super(message, cause);
        this.asyncFault = asyncFault;
    }

    public com.netsuite.webservices.platform.faults.AsyncFault getFaultInfo() {
        return this.asyncFault;
    }
}
